_episodes/01-quickstart.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Quickstart guide"
+teaching: 1
+exercises: 10
+questions:
+
+- "What is the purpose of the quickstart guide?"
+- "How do I activate and test the ESMValTool environment?"
+- "How do I configure ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Understand the purpose of the quickstart guide"
+- "Activate and test the ESMValTool environment"
+- "Configure ESMValTool"
+- "Run a recipe"
+
+keypoints:
+
+- "The purpose of the quickstart guide is to enable a user of ESMValTool
+  to run ESMValTool as quickly as possible"
+- "Use the `module load` command to activate the ESMValTool environment 
+  (see the installation guide for more details) and use `esmvaltool -- --help` to 
+  test the ESMValTool environment"
+- "Use `esmvaltool config get_config_user` to create the ESMValTool user 
+  configuration file"
+- "Use `esmvaltool run <recipe>.yml` to run a recipe"
+---
+> ## Compatibility
+>
+> This quickstart guide is compatible with ESMValTool v2.6.0.
+ {: .callout}
+
+> ## What is the purpose of the quickstart guide?
+>
+> - The purpose of the quickstart guide is to enable a user of ESMValTool to
+>   run ESMValTool as quickly as possible by making the bare minimum number of 
+>   changes.
+{: .discussion}
+
+> ## How do I activate and test the ESMValTool environment?
+>
+> - For this quickstart guide, an assumption is made that ESMValTool has 
+>   already been installed at the site where ESMValTool will be run. If this is
+>   not the case, see the [Installation][lesson-installation] episode in this
+>   tutorial.
+> 
+> - Activate the ESMValTool environment by following the instructions at
+>   [ESMValTool: Pre-installed versions on HPC clusters / other 
+>   servers][activate-environment].
+>
+> - Test the ESMValTool environment by accessing the help for ESMValTool:
+>
+>     ~~~
+>     esmvaltool -- --help
```

Comment:
When I run `esmvaltool --help` this prints out: INFO: Showing help with the command 'esmvaltool -- --help'.
So I assumed that was the correct command as when I run it that info line goes away.

Reply:
Indeed, this is the way help messages are expected by the Fire module, see https://google.github.io/python-fire/guide/. But since `esmvaltool --help` is used throughout the tutorial (e.g. [here](https://esmvalgroup.github.io/ESMValTool_Tutorial/02-installation/index.html)) and our docs (e.g. [here](https://docs.esmvaltool.org/en/latest/quickstart/installation.html)), I think it would be more consistent and user-friendly to only use that command.

Reply:
430b245 done.